src/api/api/common/index.js

```diff
@@ -25,4 +25,11 @@ module.exports = {
     return await axios.get(`${endpoints.cartsUrl}/${id}/items`)
       .then(({ data }) => data)
   },
+
+  getSvcTracker: source => {
+    return (req, type, detail = {}) =>
+      helpers.trackEvents(req, { source, events: [{type, detail}] })
+        .catch(() => { /* noop */ });
```

Comment:
Graceful event failures

Reply:
> What is the behavior if the Streaming Service is not provisioned? The tracker get ignored?

Essentially, yes. The `events` pod can enter an error state (considered normal), whereas all attempts to track from the api will fail gracefully